SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -224,7 +223,7 @@ void USpatialGDKSettings::UpdateServicesRegionFile()
 
 uint32 USpatialGDKSettings::GetRPCRingBufferSize(ERPCType RPCType) const
 {
-	if (const uint32* Size = RPCRingBufferSizeMap.Find(RPCType))
+	if (const uint32* Size = RPCRingBufferSizeOverrides.Find(RPCType))
```

Comment:
nit: AFAIU this isn't an override for the movement buffer, so would prefer keeping the name as it was

Reply:
ah wait this is separate from the movement stuff- probably fine then

Reply:
Ah I guess my way of looking at it was that the movement channel would have an override of 1, by default.